contracts/dao/proposal/cwd-proposal-multiple/src/proposal.rs

```diff
@@ -46,6 +47,22 @@ pub struct MultipleChoiceProposal {
     pub allow_revoting: bool,
 }
 
+#[cw_serde]
+pub struct OldMultipleChoiceProposal {
```

Comment:
The same about struct. It can be a local struct in the migrate handler

Reply:
This struct is utilized by the unit-test. Not sure that we'll be able to access it if the struct will be inside the migrate handler.